tests/modules/icgc-argo-workflows/score/download/test.yml

```diff
@@ -1,6 +1,6 @@
 
 - name: "score download in rdpc qa"
-  command: nextflow run ./tests/modules/icgc-argo-workflows/score/download -entry test_score_download_rdpcqa -c ./tests/config/nextflow.config -profile docker,debug_qa --api_token YOUR_TOKEN
+  command: nextflow run ./tests/modules/icgc-argo-workflows/score/download -entry test_score_download_rdpcqa -c ./tests/config/nextflow.config -profile docker,rdpc_qa --api_token YOUR_TOKEN
```

Comment:
did not find `debug_qa` profile, tests run report error
updated to `rdpc_qa`

Reply:
Unrelated but since we're cleaning up, could you also replace `debug` in other `test.yml`

https://github.com/icgc-argo-workflows/argo-modules/blob/main/tests/modules/icgc-argo-workflows/song/publish/test.yml